cyclist/src/edu/utexas/cycic/Cycic.java

```diff
@@ -540,15 +539,8 @@ public void buildDnDCircle(FacilityCircle circle, int i, String name){
 		circle.setFill(Color.rgb(circle.rgbColor.get(0),circle.rgbColor.get(1),circle.rgbColor.get(2)));
 		circle.setCenterX(45+(i*90));
 		circle.setCenterY(50);
-		circle.text.setText(name.split(" ")[2]);
-		circle.text.setWrapText(true);
-		circle.text.setMaxWidth(60);
-		circle.text.setLayoutX(circle.getCenterX()-circle.getRadius()*0.7);
-		circle.text.setLayoutY(circle.getCenterY()-circle.getRadius()*0.6);	
-		circle.text.setTextAlignment(TextAlignment.CENTER);
-		circle.text.setMouseTransparent(true);
-		circle.text.setMaxWidth(circle.getRadius()*1.4);
-		circle.text.setMaxHeight(circle.getRadius()*1.2);
+		circle.text.setText(name.split(" ")[2] + " (" + name.split(" ")[1] + ")");
+		VisFunctions.placeTextOnCircle(circle,"middle");
```

Comment:
The font color change was so I did not put black text on a dark circle. If
it is off the circle black all the time should be fine, but limits the
background colors to lighter colors.

Location looks good though!
On May 28, 2015 4:26 PM, "Matthew Gidden" <notifications@github.com> wrote:

> In cyclist/src/edu/utexas/cycic/Cycic.java
> <https://github.com/cyclus/cyclist2/pull/42#discussion_r31279546>:
>
> > @@ -540,15 +539,8 @@ public void buildDnDCircle(FacilityCircle circle, int i, String name){
> >  		circle.setFill(Color.rgb(circle.rgbColor.get(0),circle.rgbColor.get(1),circle.rgbColor.get(2)));
> >  		circle.setCenterX(45+(i*90));
> >  		circle.setCenterY(50);
> > -		circle.text.setText(name.split(" ")[2]);
> > -		circle.text.setWrapText(true);
> > -		circle.text.setMaxWidth(60);
> > -		circle.text.setLayoutX(circle.getCenterX()-circle.getRadius()*0.7);
> > -		circle.text.setLayoutY(circle.getCenterY()-circle.getRadius()*0.6);	
> > -		circle.text.setTextAlignment(TextAlignment.CENTER);
> > -		circle.text.setMouseTransparent(true);
> > -		circle.text.setMaxWidth(circle.getRadius()*1.4);
> > -		circle.text.setMaxHeight(circle.getRadius()*1.2);
> > +		circle.text.setText(name.split(" ")[2] + " (" + name.split(" ")[1] + ")");
> > +		VisFunctions.placeTextOnCircle(circle,"middle");
>
> space after comma
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclist2/pull/42/files#r31279546>.
>